eng/tools/js-sdk-release-tools/package.json

```
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@azure-tools/js-sdk-release-tools": "2.15.1"
```

@raych1 for release tool we don't want to lock the fixed version especially for patch version, we want to design this as floating version, so any patch/minor release could be taken automatically with command `npm --prefix eng/tools/js-sdk-release-tools install`. How do you think?

```suggestion
    "@azure-tools/js-sdk-release-tools": "^2.15.1"
```

---

It makes sense. You will need to update the lock file too.

---

**Behavior of package-lock.json**
The lock file’s purpose is to freeze the dependency tree, recording the exact resolved versions and the complete dependency graph.
**Even if the lock file contains something like ^1.2.3** (which normally belongs in package.json, not in the lock file), **npm will still use the specific resolved version recorded in the lock file as the installation source**.
Therefore: **if the lock file exists, npm installs according to the lock file; it does not use the ^ range to pick a newer version**.

so we cannot update the lock file.